### PR TITLE
Add Craft.io

### PR DESCRIPTION
**Pull request by muravjov1**
Add Craft.io website to Developer section

### CODE REVIEW

**Comment by RichJeanes** [comment]
Thanks for contributing! I have a few things I would like to point out for you:
1. You need to actually add the image file that you reference. Eg. Place a .png of their logo at `img/developer/craft.png`
2. The doc link is for providing access to their documentation about their 2FA implementation or, preferably, how to enable it on your account, not the company's privacy policy.
3. Could you place the entry alphabetically in the developer.yml file? It makes it easier to find and manage entries.
4. Can you explain why this fits under developer? This section is more for focusing on software development. Can craft.io be used for software? They don't seem to really provide much information about the company or product.

**Comment by muravjov1** [comment]
Hello! Thanks for feedback. Here`s my answers:
1. I don`t have access for uploading files.

[image: Встроенное изображение 1]
Can you open an access for me? I`m kind of newbie on Git..
1. Ok, that`s my mistake, we don`t have such documentation. I`ll delete
   this info.
2. Ok, I will replace it.
3. This is online tool for product and project management (Agile, Scrum
   etc.). Also you can build roadmap and push requirements with Craft. So it
   completely fits for developers. You can even register and try this tool :)

**Comment by RichJeanes** [comment]
1. If it is saying you don't have access, then you are probably trying to upload the image to our repo. You need to upload the image to the proper directory on your fork's branch. Try here: https://github.com/muravjov1/twofactorauth/upload/patch-1/img/developer
   Make sure that the file is named properly to match how it is named in your entry to developer.yml.
2. If you do get a support page set up, then feel free to come back and update this entry.

If you need further information, check out our [CONTRIBUTING.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md) and our [wiki](https://github.com/2factorauth/twofactorauth/wiki).

Also, just to clarify, craft.io supports phone calls for 2FA, but not SMS (text messaging)?

**Comment by Carlgo11** [comment]
@muravjov1 here's a smaller image. Try using that instead of the one you made.
craft io

File: _data/developer.yml [hunk 1]
```diff
@@ -97,6 +97,13 @@ websites:
       software: Yes
       doc: https://docs.compose.io/security/compose-two-factor-authentication.html
 
+    - name: Craft
+      url: https://craft.io/
+      img: craft.PNG
+      tfa: Yes
+      e-mail: Yes
```
Comment (Carlgo11) [inline _data/developer.yml:104]: This tag should be `email:` and not `e-mail:`

**Comment by Carlgo11** [comment]
You're also still missing a documentation link. Please add that.

**Comment by Carlgo11** [comment]
I just created an account on craft.io but I don't see anything mentioning any form of TFA. Could you point me to where to enable TFA if you now support it.
Also, I tried using a 100 character long password but then it said that my email had already been used. Maybe you'll need to look in to that. Seems like kind of an odd error message to me.

**Comment by RobThree** [comment]
> I just created an account on craft.io but I don't see anything mentioning any form of TFA. Could you point me to where to enable TFA if you now support it.

Agree; there's no "account settings" or similar page to be found at all. All you can do is log out.

image

There's no TFA section to be found anywhere. And it's also not possible to delete an account (if you read this @muravjov1 then please feel free to delete my account :+1: ).

**Comment by Carlgo11** [comment]
@muravjov1 will you make the proposed changes to the PR?
If not then I'll close this PR due to inactivity.

**Pull request closed without merging**